runtime/onert/backend/cpu/ops/LogisticLayer.cc

```diff
@@ -42,8 +42,27 @@ void LogisticLayer::logisticFloat32()
 
 void LogisticLayer::logisticQuant8()
 {
-  // cker quant8 logistic is not implemented yet
-  throw std::runtime_error{"NYI"};
+  uint8_t table[256];
+
+  const auto input_scale = static_cast<double>(_input->data_scale());
```

Comment:
I think table creation can be moved into `configure`.
It might reduce table creation overhead when there are multiple logistic operations in a model.

Reply:
I agree with you but
Can I add the member variable in LogisticLayer Class?
It seems that member variables in backend/cpu/ops/*.h is not defined except for input parameter of configure function.

Reply:
@intom I don't know the policy, but `FullyConnectedLayer` class has member variable `_temp_area`.